docs/pages/world/modules/keysintable.mdx

```diff
@@ -0,0 +1,239 @@
+
+import { Script } from "forge-std/Script.sol";
+import { console } from "forge-std/console.sol";
+import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
+
+import { IWorld } from "../src/codegen/world/IWorld.sol";
+import { getKeysInTable } from "@latticexyz/world-modules/src/modules/keysintable/getKeysInTable.sol";
+import { Tasks, TasksData } from "../src/codegen/index.sol";
+
+contract UseGetKeysInTable is Script {
+  function run() external {
+    uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+    address worldAddress = vm.envAddress("WORLD_ADDRESS");
+    IWorld world = IWorld(worldAddress);
+    StoreSwitch.setStoreAddress(worldAddress);
+
+    vm.startBroadcast(deployerPrivateKey);
+    world.app__addTask("Walk the cat");
+    world.app__addTask("Cook delicious meal");
+    world.app__addTask("Practice Elvish");
+    vm.stopBroadcast();
+
+    bytes32[][] memory keys = getKeysInTable(Tasks._tableId);
+
+    console.log("Number of keys:", keys.length);
+    for (uint i = 0; i < keys.length; i++) {
+      console.log("\nKey #", i, ":");
+      for (uint j = 0; j < keys[i].length; j++) {
+        console.logBytes32(keys[i][j]);
+      }
+    }
+
+    console.log("\n\nTask descriptions");
+    for (uint i = 0; i < keys.length; i++) {
+      TasksData memory task = Tasks.get(keys[i][0]);
+      console.log(task.description);
+    }
+  }
+}
+```
+
+</CollapseCode>
+
+<details>
+
+<summary>Explanation</summary>
+
+```solidity
+import { getKeysInTable } from "@latticexyz/world-modules/src/modules/keysintable/getKeysInTable.sol";
+```
+
+The function that gets the keys in the table.
+
+```solidity
+    StoreSwitch.setStoreAddress(worldAddress);
+```
+
+To use `getKeysInTable` we need to set the store address to `worldAddress`.
+
+```solidity
+    vm.startBroadcast(deployerPrivateKey);
+    world.app__addTask("Walk the cat");
+    world.app__addTask("Cook delicious meal");
+    world.app__addTask("Practice Elvish");
+    vm.stopBroadcast();
+```
+
+Creates three entries with different descriptions, to ensure we have entries to show.
+Note that this is the _only_ place in the script we need to send transactions.
+`getKeysInTable` is [a `view` function](https://docs.soliditylang.org/en/v0.8.26/contracts.html#view-functions), and does not change the state.
+
+```solidity
+    bytes32[][] memory keys = getKeysInTable(Tasks._tableId);
+```
+
+Get the key(s) by passing the Tasks table id as a parameter.
+
+```solidity
+    console.log("Number of keys:", keys.length);
+    for(uint i=0; i<keys.length; i++) {
+       console.log("\nKey #", i, ":");
+       for(uint j=0; j<keys[i].length; j++) {
+          console.logBytes32(keys[i][j]);
+       }
+    }
+```
+
+The `getKeysInTable` function returns a two-dimensional array.
+The first dimension is the keys in the table.
+The second dimension is the different key fields (in this case there is only one).
+
+```solidity
+    console.log("\n\nTask descriptions");
```

Comment:
I think its a bit easier to read if you separate the spacing from the meaningful log line
```suggestion
    console.log("\n\n");
    console.log("Task descriptions:");
```
(need to update the above script to match, too)